src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -628,7 +628,6 @@ namespace Akka.Actor
         public Deployer(Akka.Actor.Settings settings) { }
         public Akka.Actor.Deploy Lookup(Akka.Actor.ActorPath path) { }
         public Akka.Actor.Deploy Lookup(System.Collections.Generic.IEnumerable<string> path) { }
-        public Akka.Actor.Deploy Lookup(System.Collections.Generic.IEnumerator<string> path) { }
```

Comment:
Did this just appear in the API approval twice for some reason?

Reply:
no, one of them is for `IEnumerable`, the other is for `IEnumerator`, we don't need the `IEnumerator` one anymore.